crates/jstzd/src/task/octez_client.rs

```diff
@@ -124,6 +140,19 @@ impl OctezClient {
             .await?;
         Ok(())
     }
+
+    pub async fn gen_keys(
+        &self,
+        alias: &str,
+        signature: Option<Signature>,
+    ) -> Result<()> {
+        let mut command = self.command(["gen", "keys", alias])?;
+        if let Some(signature) = signature {
+            command.args(["--signature", &signature.to_string()]);
```

Comment:
Is this the correct name? I tried to run octez client with this and it gave an error and suggested
```
Commands for managing the wallet of cryptographic keys:

  gen keys <new> [-f --force] [-s --sig <ed25519|secp256k1|p256|bls>]
      [--encrypted]
    Generate a pair of keys.
    <new>: new secret_key alias
    -f --force: overwrite existing secret_key
    -s --sig <ed25519|secp256k1|p256|bls>: use custom signature algorithm
    --encrypted: Encrypt the key on-disk
```

Reply:
good catch, it's the ai tgat rewrote my code 😅